app/frontend/src/features/search/SearchResult.tsx

```diff
@@ -41,26 +55,57 @@ export default function SearchResult({ user }: { user: User.AsObject }) {
       <Card className={classes.card}>
         <CardActionArea>
           <CardContent>
-            <div className={classes.resultHeader}>
-              <Avatar user={user} />
-              <Container>
-                <Typography variant="h2">{user.name}</Typography>
-                <Typography variant="subtitle1">
+            <UserSummary user={user}>
+              <Container
```

Comment:
I wrote this `Container` a few months ago when I was young and foolish, can you get rid of it and replace it with a normal `div`

Reply:
👍👍